[2024-09-11T02:38:09.455+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-09-11T02:38:09.487+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2024-07-16T00:00:00+00:00 [queued]>
[2024-09-11T02:38:09.498+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2024-07-16T00:00:00+00:00 [queued]>
[2024-09-11T02:38:09.498+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-09-11T02:38:09.528+0000] {taskinstance.py:2330} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2024-07-16 00:00:00+00:00
[2024-09-11T02:38:09.537+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=18151) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-11T02:38:09.537+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v03', 'insert_into_table', 'scheduled__2024-07-16T00:00:00+00:00', '--job-id', '3981', '--raw', '--subdir', 'DAGS_FOLDER/13_dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmp7ya_erju']
[2024-09-11T02:38:09.539+0000] {standard_task_runner.py:91} INFO - Job 3981: Subtask insert_into_table
[2024-09-11T02:38:09.539+0000] {standard_task_runner.py:63} INFO - Started process 18152 to run task
[2024-09-11T02:38:09.592+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2024-07-16T00:00:00+00:00 [running]> on host 0359316e7bd1
[2024-09-11T02:38:09.681+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='vohoang' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v03' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2024-07-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-16T00:00:00+00:00'
[2024-09-11T02:38:09.682+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-09-11T02:38:09.697+0000] {sql.py:276} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2024-07-16', 'dag_with_postgres_operator_v03')
        
[2024-09-11T02:38:09.705+0000] {base.py:84} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-09-11T02:38:09.713+0000] {base.py:84} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-09-11T02:38:09.722+0000] {sql.py:487} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2024-07-16', 'dag_with_postgres_operator_v03')
        , parameters: None
[2024-09-11T02:38:09.725+0000] {sql.py:496} INFO - Rows affected: 1
[2024-09-11T02:38:09.729+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-09-11T02:38:09.738+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v03, task_id=insert_into_table, run_id=scheduled__2024-07-16T00:00:00+00:00, execution_date=20240716T000000, start_date=20240911T023809, end_date=20240911T023809
[2024-09-11T02:38:09.793+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-09-11T02:38:09.808+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-11T02:38:09.812+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
